index.bs

```diff
@@ -831,13 +862,20 @@ return a point in time.
 To <dfn>obtain a report</dfn> given an [=attribution source=] |source| and an [=attribution trigger=] |trigger|:
 
 1. Let |triggerData| be |trigger|'s [=attribution trigger/trigger data=].
-1. If |source|'s [=attribution source/source type=] is "<code>event</code>", set |triggerData| to |trigger|'s [=attribution trigger/event source trigger data=].
+1. Let |triggerDataNoiseProbability| be |trigger|'s [=attribution trigger/trigger data noise probability=].
+1. If |source|'s [=attribution source/source type=] is "<code>event</code>":
```

Comment:
Rather than branching on the type, can we look at the actual value of the sources noise probability and omit if it is zero? This would allow us to add noise to clicks without changing this logic at all.

Reply:
I don't think this logic will have to be changed if we add noise to clicks. This is just ensuring that the correct per-source-type trigger data and trigger data noise probability are being propagated to the report properly.

Reply:
Or are you saying that once we've determined that the source won't be attributed truthfully, that never and fake should be equally likely?

Reply:
Ah I see, I was missing some logic. We have separate trigger data / probabilities for event and nav sources in the data model. Was there a strong reason to separate those out that way?

I think thinking that this logic could be simplified if we just had single fields in `attribution trigger`s for the trigger data noise probability and the trigger data itself. These fields would be set by branching on the source type, but in report generation we could just process them as-is.

Reply:
We can't have single fields for trigger data noise probability and trigger data in the `attribution trigger` struct, because we don't know what the source type is until we've looked up matching sources, which requires the trigger itself.

Reply:
Ah you're right, that's what I was missing, thanks.